studiopress/genesis-sample
# Update screenshot.png

**Summary of change:**

Updates the screenshot for the Genesis Sample theme to reflect the OCTS content.

**Have the changes in this PR been added to the documentation for this project?**

cc @susannelson for visibility.

**How to test:**


Go to wp-admin/themes.php and view the new screenshot and verify the size is 1200x900.

**Suggested Changelog Entry:**

Changed: Updated Screenshot to reflect one-click theme setup content.
